cnd/src/btsieve.rs

```diff
@@ -12,7 +13,7 @@ pub trait MatchingTransactions<P>: Send + Sync + 'static {
     fn matching_transactions(
         &self,
         pattern: P,
-        timestamp: Option<u32>,
+        after: NaiveDateTime,
```

Comment:
`from` was the first thing I thought to use, I went with `after` because it denotes the direction whereas `from` does not - could be from now back in time or from some time in the past forward in time.  Happy to re-consider though, any other suggestions?

Reply:
Happy to leave it.